cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -116,7 +116,9 @@ spec:
         #     cpu: "500m"
         #     memory: "4Gi"
         volumeClaimTemplates:
-          - metadata:
+          - kind: VolumeClaimTemplate
+            apiVersion: v1
```

Comment:
We (and users) will have to add this to each volumeClaimTemplate.

Reply:
yeah that will be a challenge for upgraded clusters